src/utility/module.pl

```diff
@@ -168,8 +168,10 @@
         [Functor,Arity,RDF_Args,_],true) :-
   compound(RDF_Predicate),
   RDF_Predicate=..[Functor|RDF_Args],
+  subset(RDF_Args,[r,t,o,o,@,?,-,+,:]),
```

Comment:
*o* only need to be there once or?
Maybe also add a comment that these are argument properties of rdf_meta